### PR TITLE
FwLinks replacement in visual-basic folder (3)

### PR DESCRIPTION
**Pull request by florinciubotariu**
Relates to #3391 and #3426

The changes were made only in one file: visual-basic/programming-guide/concepts/async/index.md.

I looked into the equivalent C# documentation and tried to change the links and titles where it was possible. Also, some fwlinks were redirecting to old documentation so I tried to add the correct links.

### CODE REVIEW

File: docs/visual-basic/programming-guide/concepts/async/index.md [hunk 1]
```diff
@@ -230,21 +230,21 @@ Await Task_MethodAsync()
...
-|Async Cancellation: Bridging between the .NET Framework and the Windows Runtime|Shows how to bridge between Task types in the .NET Framework and IAsyncOperations in the [!INCLUDE[wrt](~/includes/wrt-md.md)] so that you can use <xref:System.Threading.CancellationTokenSource> with a [!INCLUDE[wrt](~/includes/wrt-md.md)] method.|[Async Sample: Bridging between .NET and Windows Runtime (AsTask & Cancellation)](http://go.microsoft.com/fwlink/p/?LinkId=263004)|  
+|[WhenAny: Bridging between the .NET Framework and the Windows Runtime](https://msdn.microsoft.com/library/jj635140(v=vs.120).aspx)|Shows how to bridge between Task types in the .NET Framework and IAsyncOperations in the [!INCLUDE[wrt](~/includes/wrt-md.md)] so that you can use <xref:System.Threading.Tasks.Task.WhenAny%2A> with a [!INCLUDE[wrt](~/includes/wrt-md.md)] method.|[Async Sample: Bridging between .NET and Windows Runtime (AsTask and WhenAny)](https://code.msdn.microsoft.com/Async-Sample-Bridging-d6a2f739)|
```
Comment (mairaw) [inline docs/visual-basic/programming-guide/concepts/async/index.md]: again, can you update the msdn link to https://docs.microsoft.com/previous-versions/visualstudio/visual-studio-2013/jj635140(v=vs.120)?

**Review by mairaw — CHANGES_REQUESTED** [review]
Changes look great @florinciubotariu! I just have a suggested fix and two suggestions not related to your changes. Can you please address those?

**Comment by florinciubotariu** [comment]
Hello @mairaw. Is the last commit enough for this pull-request to be accepted?

**Comment by mairaw** [comment]
Yes @florinciubotariu. I just removed the locales from the links with my last commit. Thank you so much!

**Review by mairaw — APPROVED** [review]

**Pull request merged**